### ISSUE TITLE
transmit JSON objects from pf2live to I3Live (Trac #176)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#176]
pf2live receives/requests JSON formatted component reports from JEB + PnF components. it unrolls JSON reports to single values of type boolean, integer, double and string. only single values are transmitted to I3Live since it didn't support JSON objects so far. it would be nice to have a choice between sending complete reports (which are JSON objects), report sections (again JSON objects) or single values... or maybe even in parallel.

Migrated from https://code.icecube.wisc.edu/ticket/176
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T19:21:55", 
    "description": "pf2live receives/requests JSON formatted component reports from JEB + PnF components. it unrolls JSON reports to single values of type boolean, integer, double and string. only single values are transmitted to I3Live since it didn't support JSON objects so far. it would be nice to have a choice between sending complete reports (which are JSON objects), report sections (again JSON objects) or single values... or maybe even in parallel.", 
    "reporter": "tschmidt", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547234515046259", 
    "component": "jeb + pnf", 
    "summary": "transmit JSON objects from pf2live to I3Live", 
    "priority": "normal", 
    "keywords": "JEB, I3Live, pf2live", 
    "time": "2009-11-04T16:23:10", 
    "milestone": "", 
    "owner": "tschmidt", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20100301T16:48:24: 
* blaufuss changed milestone from "" to "IC79 Physics Run"
* blaufuss changed owner from "tschmidt" to "torsten"

**Comment by icecube-trac** [comment]
Trac update at 20100301T19:44:35: blaufuss changed owner from "torsten" to "tschmidt"

**Comment by icecube-trac** [comment]
Trac update at 20120525T13:45:47: blaufuss changed milestone from "IC79 Physics Run" to ""

**Comment by icecube-trac** [comment]
Trac update at 20150211T19:53:56: 
* blaufuss changed status from "new" to "closed"
* blaufuss changed resolution from "" to "invalid"

**Comment by icecube-trac** [comment]
Trac update at 20190111T18:55:29: 
* nwhitehorn changed resolution from "invalid" to "fixed"
* nwhitehorn commented:

> In changeset 167891:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167891"
> Don't automatically purge before merging.  At this point I think the
> interface does everything we'd need.
> 
> Fixes #176 along with some earlier commits.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T19:21:55: nwhitehorn commented:

> In changeset 168029:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168029"
> Don't automatically purge before merging.  At this point I think the
> interface does everything we'd need.
> 
> Fixes #176 along with some earlier commits.
> ```
